jasper_server/common/__init__.py

```diff
@@ -0,0 +1,20 @@
+# b-*- encoding: utf-8 -*-
```

Comment:
Yes
Actually, I don't know why, but without this file, imports of files from this folder do not work:

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 180, in run_wsgi
    execute(self.server.app)
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 168, in execute
    application_iter = app(environ, start_response)
  File "/home/esal1/Gattlen_v10/git_repositories/odoo/odoo/service/server.py", line 246, in app
    return self.app(e, s)
  File "/home/esal1/Gattlen_v10/git_repositories/odoo/odoo/service/wsgi_server.py", line 184, in application
    return application_unproxied(environ, start_response)
  File "/home/esal1/Gattlen_v10/git_repositories/odoo/odoo/service/wsgi_server.py", line 170, in application_unproxied
    result = handler(environ, start_response)
  File "/home/esal1/Gattlen_v10/git_repositories/odoo/odoo/http.py", line 1305, in __call__
    self.load_addons()
  File "/home/esal1/Gattlen_v10/git_repositories/odoo/odoo/http.py", line 1326, in load_addons
    m = __import__('odoo.addons.' + module)
  File "/home/esal1/Gattlen_v10/git_repositories/odoo/odoo/modules/module.py", line 81, in load_module
    execfile(modfile, new_mod.__dict__)
  File "/home/esal1/Gattlen_v10/git_repositories/openerp-jasperserver/jasper_server/__init__.py", line 25, in <module>
    import models
  File "/home/esal1/Gattlen_v10/git_repositories/openerp-jasperserver/jasper_server/models/__init__.py", line 23, in <module>
    import jasper_server
  File "/home/esal1/Gattlen_v10/git_repositories/openerp-jasperserver/jasper_server/models/jasper_server.py", line 29, in <module>
    from odoo.addons.jasper_server.common import jasperlib
ImportError: No module named common